### PR TITLE
Openstack Attach Detach Volume Actions

### PR DESCRIPTION
**Pull request by pgorgolew**

### CODE REVIEW

File: executor/src/main/java/org/k8loud/executor/action/openstack/AttachVolumeAction.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    public AttachVolumeAction(Params params, OpenstackService openstackService) throws ActionException {
+        super(params, openstackService);
+    }
+
+    @Override
+    public void unpackParams(Params params) {
+        region = params.getRequiredParam("region");
+        serverId = params.getRequiredParam("serverId");
+        volumeId = params.getRequiredParam("volumeId");
```
Comment (proman3419) [inline executor/src/main/java/org/k8loud/executor/action/openstack/AttachVolumeAction.java:22]: Maybe move common fields declaration and initialization to a `VolumeAction`  or `AttachDetachVolumeAction` class and extend it?

File: executor/src/main/java/org/k8loud/executor/openstack/OpenstackService.java [hunk 2]
```diff
@@ -3,14 +3,15 @@
 import org.k8loud.executor.exception.OpenstackException;
 import org.k8loud.executor.util.annotation.ThrowExceptionAndLogExecutionTime;
 
+//FIXME enable annotation here not in implementation (with spring-retry working)
```
Comment (proman3419) [inline executor/src/main/java/org/k8loud/executor/openstack/OpenstackService.java:6]: Story?

**Review by proman3419 — APPROVED** [review]
One thing to consider, either way is okay

File: executor/src/main/java/org/k8loud/executor/openstack/OpenstackService.java (hunk 2 above)
Comment (pgorgolew) [reply executor/src/main/java/org/k8loud/executor/openstack/OpenstackService.java:6]: added

**Pull request merged**